R/app_server.R

```diff
@@ -6,7 +6,7 @@
 #' @noRd
 app_server <- function(input, output, session) {
   # Your application server logic
-  
+  mod_taxonomy_server("taxonomy_1")
```

Comment:
Can this tab go after the other two for now? I think that's probably where it will end up in future anyway.